base/tks/src/main/java/org/dogtagpki/server/tks/servlet/TokenServlet.java

```diff
@@ -2815,10 +2883,28 @@ private void processComputeSessionKeysSCP03(HttpServletRequest req, HttpServletR
         }
 
         if (serversideKeygen == true) {
-            if (serverSideValues.size() == 3) {
+            logger.debug(method + " serversideValues.size: " + serverSideValues.size());
+	    for(int i = 0 ; i < serverSideValues.size(); i++) {
+                logger.debug(method + " val:" + i  + " : " + serverSideValues.get(i));
```

Comment:
I think you want to comment out these few debug lines

Reply:
I think it's ok to print out the size might be useful when debugging an issue. Commenting out the ones that actually print the values.